src/topicButton.js

```diff
@@ -1,9 +1,22 @@
 /* exported TopicButton */
-
-const {GObject, Gtk} = imports.gi;
+const {GObject, Gtk, GLib, Gio} = imports.gi;
 
 const {Lockscreen} = imports.lockscreen;
 
+const TopicIface = `
+<node>
+  <interface name="com.endlessm.Toolbox.topic">
```

Comment:
Early-tester review: I think the interface name should be: com.endlessm.**Hack**Toolbox.topic (to match the app name).

Reply:
Done, thanks!